### PR TITLE
Fix message edits

### PR DESCRIPTION
**Pull request by tbarbugli**
Fixes an issue where message cells are not updating correctly their size

### CODE REVIEW

File: DemoApp/DemoAppCoordinator.swift [hunk 1]
```diff
@@ -74,7 +74,7 @@ final class DemoAppCoordinator: NSObject, UNUserNotificationCenterDelegate {
         
         // Create client
         var config = ChatClientConfig(apiKey: .init(apiKeyString))
-        config.isLocalStorageEnabled = true
+//        config.isLocalStorageEnabled = true
```
Comment (nuno-vieira) [inline DemoApp/DemoAppCoordinator.swift:77]: uncomment this please

File: DemoAppPush/NotificationService.swift [hunk 2]
```diff
@@ -78,7 +78,7 @@ class NotificationService: UNNotificationServiceExtension {
         }
 
         var config = ChatClientConfig(apiKey: .init(apiKeyString))
-        config.isLocalStorageEnabled = true
+//        config.isLocalStorageEnabled = true
```
Comment (nuno-vieira) [inline DemoAppPush/NotificationService.swift:81]: uncomment this please

**Comment by nuno-vieira** [comment]
Remove the forgotten commented code + Changelog and then 🚢 ✅

File: DemoApp/DemoAppCoordinator.swift (hunk 1 above)
Comment (tbarbugli) [reply DemoApp/DemoAppCoordinator.swift:77]: its done on purpose ;)

**Pull request merged**